Dockerfile

```
@@ -99,15 +99,18 @@ FROM production as dev
 # Swap user, so the following tasks can be run as root
 USER root
 
-# Install node (Keep the version in sync with the node container above)
-RUN curl -fsSL https://deb.nodesource.com/setup_14.x | bash - && apt-get install -y nodejs
-
 # Install `psql`, useful for `manage.py dbshell`
 RUN apt-get install -y postgresql-client
```

I got a `0.080 E: Unable to locate package postgresql-client` error on `fab build`. The following should fix it.

```suggestion
RUN apt-get update --yes --quiet && apt-get install --yes --quiet --no-install-recommends \
    postgresql-client \
    && apt-get autoremove && rm -rf /var/lib/apt/lists/*
```